snapshots/btrfs/btrfs_test.go

```diff
@@ -29,20 +29,23 @@ func boltSnapshotter(t *testing.T) func(context.Context, string) (snapshots.Snap
 
 	return func(ctx context.Context, root string) (snapshots.Snapshotter, func() error, error) {
 
-		deviceName, cleanupDevice, err := testutil.NewLoopback(100 << 20) // 100 MB
+		deviceName, cleanupDevice, err := testutil.NewLoopback(650 << 20) // 650 MB
```

Comment:
I can leave a code comment here if you want, but this was the minimum size I could get btrfs to work in a container on `ppc64le`. I tested with `s390x` and it worked with an even smaller value. I don't have the right hardware to test this with arm*

Reply:
Why the required size significantly differs across architectures?
Is there ppc-specific bug in our code or in kernel?

Reply:
I couldn't find any information / bugs reported on this, but it's definitely a kernel issue and not a containerd one.

Reply:
Could you open a github issue?

Reply:
Please fix this for systems that work under this size. This is way too big to be allocating on small hosts.